pages/api/user/index.ts

```diff
@@ -1,7 +1,13 @@
 import { NextApiRequest, NextApiResponse } from 'next';
-import { errorResponse, successResponse } from '../../../lib/http.response';
-import { prisma } from '../../../lib/prisma';
+import { prisma } from '@/lib/prisma';
 import { User } from '@prisma/client';
+import bodyValidator from '@/lib/bodyValidator';
+import {
+  errorResponse,
+  successResponse,
+  validationResponse,
+} from '@/lib/http.response';
+import { userSchema } from '@/schema/userSchema';
```

Comment:
```suggestion
import { userSchema } from '@/schema';
```

Reply:
<img width="826" alt="image" src="https://user-images.githubusercontent.com/60838584/190154559-bc45357b-73a3-46ad-854c-75e807b7dcfc.png">

Reply:
could you try @/schema/index?

Reply:
this way it does work `import { postSchema } from '@/schema/index';`

Reply:
then use like this. later we can look at this issue. benefit is we get all schema from one import only.